pages/index.js

```diff
@@ -1,11 +1,11 @@
 import Header from "@/components/Header/Header";
 import TaskList from "@/components/TaskList/TaskList";
 
-export default function HomePage() {
+export default function HomePage({ sortedDefaultTasks }) {
   return (
     <main>
       <Header />
-      <TaskList />
+      <TaskList sortedDefaultTasks={sortedDefaultTasks} />
```

Comment:
I suggest to move the `header` out of the `main` element.

**`<header>`:** The header section of the page, which typically contains navigation links, logos, and headings.
**`<main>`:** The main content area of the page. It contains the primary content that users see and interact with. It is important for accessibility and SEO because it helps search engines and screen readers identify the main content of the page.